bpo-36540 What's New section for PEP 570 (#12942)
python/cpython: Doc/whatsnew/3.8.rst

.. TODO: Emily will sprint on docs at PyCon US 2019.

Positional-only parameters
--------------------------

> There is new syntax (``/``) to indicate that some function parameters

Reviewer: ```suggestion
New syntax (``/``) has been added to indicate that some function parameters
```

"There is new syntax" also is fine grammatically, but usually the voice of these "What's new" entries seems to assume the context is that you are explaining what this PEP changes. "There is new syntax" feels every so slightly off because it de-emphasizes the change aspect of it.

I'll note that to me this is a very subtle difference and if I were not reading this specifically with giving notes on the wording in mind I would probably not have batted an eye, so if you prefer the original wording that's fine.
